sources/core/Stride.Core/Storage/ObjectId.cs

```diff
@@ -95,7 +95,7 @@ public static void Combine(ref ObjectId left, ref ObjectId right, out ObjectId r
         }
 
         /// <summary>
-        /// Performs an explicit conversion from <see cref="ObjectId"/> to <see cref="byte[]"/>.
+        /// Performs an explicit conversion from <see cref="ObjectId"/> to <see cref="byte"/>[].
```

Comment:
`<see cref="T:byte[]"/>` doesn't work ?

Reply:
no that doesnt work,
<see cref="byte[]"/> is understandable for Visual Studio , but it still just shows byte type
docfx cant handle it completely
![grafik](https://github.com/stride3d/stride/assets/107197024/0532a492-9deb-4c20-b579-3db7cb91b3a5)
with the generic type it doesnt work for visual studio at all
![grafik](https://github.com/stride3d/stride/assets/107197024/6accd9db-3374-4422-bcbe-f4f3ea1f2cf3)